examples/webserver.rs

```diff
@@ -1,36 +1,37 @@
-use actix_web::{web, App, HttpResponse, HttpServer, Responder};
-use identicon_rs::{Identicon, ImageType};
+use actix_web::{web, App, HttpRequest, HttpResponse, HttpServer, Responder};
+use identicon_rs::Identicon;
 
-fn generate_png(path_input: web::Path<String>) -> impl Responder {
-    let identicon = Identicon::new_default(&path_input);
-    let file = identicon.export_file_data(ImageType::PNG);
+async fn generate_png(req: HttpRequest) -> impl Responder {
+    let identicon_string = req.match_info().get("name").unwrap();
+    let identicon = Identicon::new(identicon_string);
+    let file = identicon.export_png_data().unwrap();
 
     HttpResponse::Ok().content_type("image/png").body(file)
 }
 
-fn generate_jpeg(path_input: web::Path<String>) -> impl Responder {
-    let identicon = Identicon::new_default(&path_input);
-    let file = identicon.export_file_data(ImageType::JPEG);
+async fn generate_jpeg(req: HttpRequest) -> impl Responder {
+    let identicon_string = req.match_info().get("name").unwrap();
+    let identicon = Identicon::new(identicon_string);
+    let file = identicon.export_jpeg_data().unwrap();
 
     HttpResponse::Ok().content_type("image/jpeg").body(file)
 }
 
-fn main() {
-    let address = "[::1]:8088";
+#[actix_rt::main]
+async fn main() -> std::io::Result<()> {
+    let address = "127.0.0.1:8088";
```

Comment:
I would stay on IPv6
```suggestion
    let address = "[::1]:8088";
```